ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/CommandGenerator.scala

```diff
@@ -24,65 +24,70 @@ final class CommandGenerator(
       .toMap
   private val observersWithIndices: List[(Primitive.Party, Int)] = observers.zipWithIndex
 
-  def next(): Try[Primitive.Party => Command] =
+  def nextSingle(): Try[Command] =
```

Comment:
This isn't used anywhere. Are we going to support both batched and single modes in the future?